### ISSUE TITLE
evaluator

### ISSUE THREAD

**Issue by wengrx** [microsoft/MASS#34]
At the evaluation process, do you need to mask the input sentences? I don't find this operation in the code. So the PPL reported in the paper is computed when the input is a whole sentence?

**Comment by hyeseonko** [comment]
+1 same question

**Comment by tobyoup** [comment]
Masking is only conducted in the pre-training stage. For fine-tuning and evaluation, it follows the practice in the downstream tasks.

**Comment by StillKeepTry** [comment]
@wengrx @hyeseonko I have updated the evaluation code for calculating the ppl of the masked segment. [This](https://github.com/microsoft/MASS/blob/67d25a9959e3712042a081a830bf2832212e479e/MASS/src/evaluation/evaluator.py#L355) may be your want.

**Comment by wengrx** [comment]
Thanks very much.
